src/components/sections/Catalogue/CurrentProductSection/CurrentProductSection.tsx

```
@@ -26,6 +27,16 @@ export default function CurrentProductSection({
     queryFn: async () => await getProductBySlug(slug)
   })
 
+  const {
+    data: reviews,
+    isLoading: isReviewsLoading,
+    error: reviewsError
+  } = useQuery<any[]>({
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)